src/pages/MainPage.jsx

```diff
@@ -28,43 +29,53 @@ const photos = [
   'https://images.unsplash.com/photo-1617505907947-9cb31eb80183?ixid=MXwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHw%3D&ixlib=rb-1.2.1&auto=format&fit=crop&w=1867&q=80',
 ];
 
+let user;
+
```

Comment:
Instead of declaring the user here, you'll want to use
```js
const [getUser, setUser] = useState(null);
```

Then when the response returns
```js
setUser(res);
```

This is because when you refresh the page, `MainPage` gets the user, but because it's not kept in its state, it won't be passed down to the children when the value of user updates.

Reply:
Gotcha